slices/libx11-data.yaml

```
@@ -0,0 +1,78 @@
+package: libx11-data
+
+essential:
+  - libx11-data_copyright
+
+slices:
+  data:
+    contents:
+      /usr/share/X11/XErrorDB:
```

```suggestion
  data:
    essential:
      - libx11-data_locale
    contents:
      /usr/share/X11/XErrorDB:
      
  locale:
    contents:
```

just because we have some other SDFs where we design a specific `locale` slice